packages/mobile/src/exchange/ExchangeReview.tsx

```diff
@@ -172,63 +194,115 @@ export class ExchangeReview extends React.Component<Props, State> {
                     tokenName: t(`global:${this.state.inputTokenDisplayName}`),
                   })}
                 </Text>
-                <Text style={styles.currencyAmountText}>
-                  {getMoneyDisplayValue(this.state.inputAmount, this.state.inputToken, true)}
-                </Text>
+                <CurrencyDisplay
+                  style={styles.currencyAmountText}
+                  amount={{
+                    value: this.state.inputAmount.toString(),
+                    currencyCode: CURRENCIES[this.state.inputToken].code,
```

Comment:
Can this be simplified? E.g. CurrencyDisplay takes the currency and finds the code itself?

Reply:
Yes, this will be simplified in a future PR where we merge/refactor `CURRENCY_ENUM` and `LocalCurrency`.